InvenTree/part/views.py

```diff
@@ -719,6 +724,168 @@ def get_initial(self):
         return initials
 
 
+class PartImport(FileManagementFormView):
+    ''' Part: Upload file, match to fields and import parts(using multi-Step form) '''
+    permission_required = 'part.add'
+
+    class PartFileManager(FileManager):
```

Comment:
Move to `InvenTree/common/files.py` for re-use?

Reply:
see previous comment about context